task-launcher/src/tasks/shared/trials/finishExperiment.js

```diff
@@ -1,12 +1,29 @@
 import { jsPsych } from "../../taskSetup";
-import { taskStore } from "../helpers";
+import { taskStore, PageAudioHandler } from "../helpers";
+import { mediaAssets} from '../../..'; 
 
 
 export function finishExperiment() {
     const t = taskStore().translations;
     jsPsych.endExperiment(
-      `<div id="prompt-container-text">
-                <p id="prompt">${t?.taskCompletion || 'Thank you for completing the task! You can now close the window.'}</p>
-            </div>`,
-    ); // ToDo: style text and add audio message?
+        `<div class='instructions-container'>
+            <div class='lev-row-container instruction'>
+                <h1 class='instructions-title'>${t.taskFinished}</h1>
+            </div>
+            <footer>${t.generalFooter}</footer>
+            <button id="exit-button" class="primary" style=margin-top:5%>Exit</button>
+        </div>`,
+        PageAudioHandler.playAudio(mediaAssets.audio.taskFinished)
+    ); 
+
+    window.addEventListener('click', (event) => {
+        const buttonId = event.target.id;
+        if (buttonId === 'exit-button') {
```

Comment:
I forgot to mention that we should probably also remove the listener. So my suggestion would be to move this listener to before `jsPsych.endExperiment` is declared and remove the listener once the button is placed.
```
const removeDOMElements = (event) => {
        const buttonId = event.target.id;
        if (buttonId === 'exit-button') {
           document.body.innerHTML = '';
            // clean up so that it does not persist
           window.removeEventListener('click', removeDOMElements);
        }
}
window.addEventListener('click', removeDOMElements);
```